.docker/app/nginx.conf

```diff
@@ -0,0 +1,33 @@
+user nginx;
+worker_processes auto;
+
+events {
+    worker_connections 4096;
+}
+
+http {
+    log_format combined_extra '$http_x_forwarded_for - $remote_user [$time_local] '
+                              '"$request" $status $bytes_sent '
+                              '"$http_referer" "$http_user_agent" '
+                              '$request_time';
+
+    access_log /dev/stdout combined_extra;
+    error_log /dev/stderr warn;
+
+    server {
+        listen      80;
+        server_name localhost _;
+        root        /app/public;
+
+        gzip on;
+        gzip_comp_level 6;
+        gzip_types *;
+
+        location / {
+            expires 60s;
```

Comment:
This is like a timeout?

Reply:
This is cache-related setting, better remove it. Thanks for pointing this out :mag:

Reply:
Nope, it's the `Expires` header set on the response, for caching purposes on the client.
cf https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expires

Actually, you could even set immutable cache rule if you change the name of the delivered asset (ie by adding an hash to it). But it is not mandatory for now.

Reply:
> better remove it

Actually you can let it, it'll make the item cacheable on the client for 60s.